src/components/AppRoot/AppRoot.tsx

```diff
@@ -72,10 +72,13 @@ export const AppRoot = withAdaptivity<AppRootProps>(
 
     if (process.env.NODE_ENV === "development") {
       if (scroll !== "global" && mode !== "embedded") {
-        warn("Scroll modes only supported in embedded mode", "error");
+        warn(
+          `scroll=${scroll} поддерживается только в embedded режиме`,
+          "error"
+        );
       }
       if (_mode && _embedded) {
-        warn(`mode="${mode}" overrides embedded`);
+        warn(`mode="${mode}" отменяет embedded режим`);
```

Comment:
Здесь все-таки не совсем про отмену, а про то, что если `mode` и `embedded` переданы одновременно, то `mode` победит.

```suggestion
        warn(`Свойство "mode=${mode}" приоритетнее, чем "embedded"`);
```